### ISSUE TITLE
Add prerender to Tutorial

### ISSUE THREAD

**Issue by cannikin** [redwoodjs/graphql#4812]
It's so awesome and everyone will want to use it, ~let's introduce it from the start! Before deploy and probably before Auth?~

Thinking about this some more and I think it'll be better to put in part 2 or even 3 of the tutorial. Part 3 was intended to be the **optimization** part, of which prerender is a huge part.

**Comment by jtoar** [comment]
@cannikin This didn't make it into the tutorial right? Are you still thinking about it or should we close this for now?

**Comment by cannikin** [comment]
Yeah let's hold off for now. Once we get around to more caching stuff and/or SSR it's probably time for a part 3 (or chapters 8, 9, 10) and we can cover this. I could just throw it into the end but I realized there's more to worry about, like cells and an explanation of how those work, what is and isn't prerenderable, etc.
